pkg/backingstore/reconciler.go

```diff
@@ -761,18 +793,28 @@ func (r *Reconciler) ReconcileExternalConnection() error {
 // ReconcilePool handles the pool using noobaa api
 func (r *Reconciler) ReconcilePool() error {
 
-	// TODO we only support creation here, but not updates
 	if r.PoolInfo != nil {
-		return nil
+		if r.ScaleHostsPoolParams != nil {
+			err := r.NBClient.ScaleHostsPoolAPI(*r.ScaleHostsPoolParams)
+			if err != nil {
+				return err
+			}
+		}
+		return r.reconcilePvPool()
 	}
 
 	poolName := ""
 
 	if r.CreateHostsPoolParams != nil {
-		err := r.NBClient.CreateHostsPoolAPI(*r.CreateHostsPoolParams)
+		res, err := r.NBClient.CreateHostsPoolAPI(*r.CreateHostsPoolParams)
```

Comment:
same as before. I think that it's better if the actual values are being read by noobaa-core directly from the cluster.

Reply:
for now as discussed we will keep - as 2 different creates suppose to be impossible